cranelift/codegen/src/machinst/isle.rs

```diff
@@ -18,6 +19,8 @@ pub type WritableReg = Writable<Reg>;
 pub type VecReg = Vec<Reg>;
 pub type VecWritableReg = Vec<WritableReg>;
 pub type ValueRegs = crate::machinst::ValueRegs<Reg>;
+pub type InstOutput = SmallVec<[ValueRegs; 2]>;
+pub type InstOutputBuilder = Cell<InstOutput>;
```

Comment:
I don't see any uses of this builder in the current PR; is it required for some later changes, e.g. handling calls with a variable number of returns? (That's fine if so; I just want to make sure it's not a leftover.)

Reply:
Yes, exactly.   This is used in https://github.com/bytecodealliance/wasmtime/pull/3785 here to construct the output for call instructions:
https://github.com/bytecodealliance/wasmtime/pull/3785/files#diff-1eac808500791b38886e17f448d60faa6583f025fef1c810fadc9dfdc04efe37R2275

Reply:
Makes sense, thanks!